__tests__/app.test.js

```diff
@@ -4,6 +4,7 @@ const app = require("../app.js");
 const db = require("../db/connection");
 const seed = require("../db/seeds/seed.js");
 const data = require("../db/data/test-data/index.js");
+require("jest-sorted");
```

Comment:
You shouldn't need to require in jest sorted - we actually need to add the set up code to our package.json - you can find this in the jest sorted docs!